server/s3_error_codes.cc

```diff
@@ -53,6 +57,12 @@ std::string& S3Error::to_xml(bool no_decl) {
     xml_message +=
         S3CommonUtilities::format_xml_string("Message", auth_error_message);
   }
+  for (auto arg : invalid_args) {
```

Comment:
Please post a link of amazon s3 documentation regarding including the invalid arguments in the error response.

Reply:
I could not find any document that describes error response in detail. This is purely based on observation.

Reply:
ok, please post the observation for info.

Reply:
We should be documenting all of these kinds of observations somewhere. Not sure if in LLD, or a general document of errors.

Reply:
Updated the observation on page - https://seagate-systems.atlassian.net/wiki/spaces/PUB/pages/817889369/Observations